client/src/components/Profile/StudentStatsCard.tsx

```diff
@@ -146,7 +147,7 @@ class StudentStatsCard extends React.Component<Props, State> {
                     <p style={{ fontSize: 12, marginBottom: 5 }}>
                       Score: <Text mark>{totalScore}</Text>
                     </p>
-                    {!(isExpelled || isCourseCompleted) ? (
+                    {!(isExpelled || isCourseCompleted) && isProfileOwner ? (
```

Comment:
suggest to change to the following line. It will require less mental work to evaluate condition (at least for me)

```suggestion
                    {(!isExpelled && !isCourseCompleted) || isProfileOwner ? (
```

Reply:
Why not